Script.js

```diff
@@ -257,3 +257,33 @@ const hidepop = document.getElementsByClassName('closepop');
 for (let i = 0; i < hidepop.length; i += 1) {
   closewindow(hidepop[i], i);
 }
+
+/// validation
+let timeout;
+const formval = document.getElementById('formcont');
+formval[1].addEventListener('keydown', () => {
+  clearTimeout(timeout);
+  timeout = setTimeout(() => {
+    console.log('Has dejado de escribir en el input');
+    clearTimeout(timeout);
+  }, 1000);
+});
+formval.addEventListener('submit', (event) => {
+  function mostrar(value) {
+    const message = document.getElementById('error');
+    const emailadd = value;
+    const check = {
+      lower: /^[a-z0-9._-]+@[a-z0-9.-]+\.([a-z]{2,4})+$/,
+    };
+    if (check.lower.test(emailadd) === false) {
+      event.preventDefault();
+      message.style.display = 'flex';
+    } else {
+      // event.preventDefault()
```

Comment:
- [ ] Kindly consider removing commented-out codes from your codebase to make your code look cleaner and easy to read.